docs/releases/2.0.txt

```diff
@@ -140,7 +140,9 @@ CSRF
 Database backends
 ~~~~~~~~~~~~~~~~~
 
-* ...
+* The new ``DatabaseWrapper.display_name`` attribute allows specifying the
```

Comment:
I'd put this in "backwards incompatible changes" instead (not really incompatible but it's a change that backends should make): 
```
* Add a ``DatabaseWrapper.display_name`` attribute with the name of the
  database that your backend works with. Django may use it in various messages,
  such as in system checks.
```